src/views/about.html

```diff
@@ -49,6 +51,32 @@ <h4><%=officer.position%></h4>
                 <% } %>
         </div></div>
     </section>
+
+    <section>
+        <h1>Former Officers</h1>
+        <div class="row">
+            <div class="row col-lg-6">
+                <% var i = 0 %>
+                <% for (const officer of former) { %>
+                    <div class="officer-card col-lg-6 col-md-6 col-sm-6">
+                        <div class="officer-img" style="background-image: url('<%=officer.image%>')"></div>
+                        <h3><%=officer.name%></h3>
+                        <h4><%=officer.major%></h4>
+                        <h4><%=officer.position%></h4>
+                    </div>
+                <% i++; %>
+                    <% if (i == 2) { %>
+                        </div><div class="row col-lg-6">
+                    <% } %>
+                    <% if (i == 4) { %>
+                        </div></div><div class="row"><div class="row col-lg-6">
+                        <% i = 0 %>
```

Comment:
Can you explain this part?

Reply:
That is for when more former officers will be added. Currently, there is just one. Is that incorrect?